lib/promscrape/discovery/gce/instance.go

```diff
@@ -160,8 +160,10 @@ func (inst *Instance) appendTargetLabels(ms []*promutils.Labels, project, tagSep
 	for _, item := range inst.Metadata.Items {
```

Comment:
Do other objects require the same check?

Reply:
No, those objects are passed by value unlike Labels which uses `*promutils.Labels`  starting from 1.85.3.
I've checked other discovery methods to see if same issue will happen for another methods, most discovery methods are using `map[string]string` which won't result into errors.